backend/variants/tests/test_queries.py

```diff
@@ -2036,22 +2037,22 @@ def test_exon_dist_one_vcf(self):
         self.run_query(CasePrefetchQuery, {"max_exon_dist": 1}, 1)
 
     def test_exon_dist_two_filter(self):
-        self.run_query(CasePrefetchQuery, {"max_exon_dist": 10}, 2)
+        self.run_query(CasePrefetchQuery, {"max_exon_dist": 10}, 3)
```

Comment:
why the change from 2 to 3 and 3 to 4?

Reply:
ah that's the number of cases w.r.t filtering

Reply:
The third parameter is the number of results that is expected. I added a variant to the test set with exon dist `-10` to test the absolute function in the filter. As the max exon dist is set to `10`, it should return also the one with exon dist `-10`, making it 3 instead of 2 variants (i.e. exon dist `1`, `10` and `-10`).